### PR TITLE
Add scaffolding for tizen native example app

### PR DESCRIPTION
**Pull request by zhoonit**
This PR adds an anchor point for tizen native example app.
More specifically contains a tizen wearable widget that prints out 'hello world'
I am planning to build a tizen wearable app that trains user drawing labeled with a specific unicode character but subjected to change at the moment.

Self evaluation:

Build test: [X]Passed [ ]Failed [ ]Skipped
Run test: [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: Jihoon Lee <jhoon.it.lee@samsung.com>

### CODE REVIEW

**Comment by myungjoo** [comment]
Please add ```README.md``` at  Applications/Tizen_native/CustomShortcut/
Describe what this app does and a short how-to-build/execute (which emulator, which profile and so on)
Then, please attach a screenshot! 🎈

**Comment by zhoonit** [comment]
> Please add `README.md` at Applications/Tizen_native/CustomShortcut/
> Describe what this app does and a short how-to-build/execute (which emulator, which profile and so on)
> Then, please attach a screenshot! 🎈

I added a brief `README.md`

I will elaborate more with some images as things get sorted out :)

**Comment by jijoongmoon** [comment]
Do we need all the .(dot) files? could you check this again? are they you create or generated by the compiler?

**Comment by zhoonit** [comment]
> Do we need all the .(dot) files? could you check this again? are they you create or generated by the compiler? If it is generated by compiler.. then we don't need to here.

IMO Yes.

1. I initialized with `tizen studio template` and template contains those files. These are not generated by the compiler though.
2. Build fails without those file while these are not generated from `tizen build-native`
3. As for that reason, `tizen clean` does not delete those files away. They only delete `/Debug/`
4. Other example repos contains those files.
 eg) 
 https://github.com/Samsung/tizen-extension-sample/tree/master/Native/InAppPurchase
 https://github.com/Samsung/tizen-fundamental-classes/tree/master/TizenFundamentalClasses
 https://github.com/nnstreamer/nnstreamer-example/tree/master/Tizen.native/ObjectDetection

I think if we are going to make a build/packaging script, we may or may not need those files. 
However, since this is an example app for others can easily try with there tizen studio, I think it would be better to leave it as it is :)

File: Applications/Tizen_native/CustomShortcut/src/main.c [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+typedef struct widget_instance_data {
+	Evas_Object *win;
+	Evas_Object *conform;
+	Evas_Object *label;
+} widget_instance_data_s;
+
+static int
+widget_instance_create(widget_context_h context, bundle *content, int w, int h, void *user_data)
+{
+	widget_instance_data_s *wid = (widget_instance_data_s*) malloc(sizeof(widget_instance_data_s));
```
Comment (again4you) [inline Applications/Tizen_native/CustomShortcut/src/main.c:22]: All Tizen native apps as well as nnstreamer use 2 spaces indentation instead of Tab.

Comment (zhoonit) [reply Applications/Tizen_native/CustomShortcut/src/main.c]: Fixed. Thanks 😃

File: Applications/Tizen_native/CustomShortcut/src/main.c [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+ */
+
+#include "main.h"
+#include "view.h"
+#include <tizen.h>
+
+static int widget_instance_create(widget_context_h context, bundle *content,
+                                  int w, int h, void *user_data) {
+  widget_instance_data_s *wid =
+    (widget_instance_data_s *)malloc(sizeof(widget_instance_data_s));
```
Comment (dongju-chae) [inline Applications/Tizen_native/CustomShortcut/src/main.c:17]: need error handling here

File: Applications/Tizen_native/CustomShortcut/src/main.c [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+  elm_object_text_set(wid->label, "Hello widget!");
+
+  return WIDGET_ERROR_NONE;
+}
+
+static int widget_instance_destroy(widget_context_h context,
+                                   widget_app_destroy_type_e reason,
+                                   bundle *content, void *user_data) {
+  widget_instance_data_s *wid = NULL;
+  widget_app_context_get_tag(context, (void **)&wid);
```
Comment (dongju-chae) [inline Applications/Tizen_native/CustomShortcut/src/main.c:46]: Please check the return value of `widget_app_context_get_tag ()`

File: Applications/Tizen_native/CustomShortcut/src/main.c (hunk 2 above)
Comment (zhoonit) [reply Applications/Tizen_native/CustomShortcut/src/main.c:18]: Thanks. fixed!

File: Applications/Tizen_native/CustomShortcut/src/main.c (hunk 3 above)
Comment (zhoonit) [reply Applications/Tizen_native/CustomShortcut/src/main.c]: Thank you, I checked the official docs and handled it :)

**Pull request merged**